dayrepo/snippets/static/js/process_add.js

```diff
@@ -0,0 +1,9 @@
+var i = 1 ;
```

Comment:
```suggestion
let i = 1 ;
```

現代のJavaScript では、基本的に var で変数を宣言することは非推奨です。
なぜ非推奨なのか？の背景は重要なので、以下の参考記事に目を通しておいてくださいmm 
FYI
[JavaScriptでvarが非推奨な理由を整理してみた](https://qiita.com/ist-t-s/items/49ed558bd837452353b8)
[varはもう使わない](https://typescriptbook.jp/reference/values-types-variables/vars-problems)